warp/builtins.py

```
@@ -4795,41 +4795,72 @@ def compute():
     :param id: The BVH identifier
     :param low: The lower bound of the bounding box in BVH space
-    :param high: The upper bound of the bounding box in BVH space""",
+    :param high: The upper bound of the bounding box in BVH space
+    :param root: The root to begin the query from (optional, default: -1)""",
     export=False,
 )
 
 add_builtin(
     "bvh_query_ray",
-    input_types={"id": uint64, "start": vec3, "dir": vec3},
+    input_types={"id": uint64, "start": vec3, "dir": vec3, "root": int, "max_dist": float},
```

Ruff (0.13.1)

`uint64` may be undefined, or defined from star imports

(F405)